frontend/css/theme.css

```
@@ -20,4 +20,57 @@
     background-color: transparent;
     border: none;
     color: white;
-}
\ No newline at end of file
+}
+
+/* APPOINTMENT */
+
+.appointment {
+    margin-bottom: 5px;
+    margin-left: 20%;
+    margin-right: 20%;
+    text-align: center;
+    border-style: solid;
+    border-radius: 15px 50px;
+    border-color: blue;
+    
+}
+
+.noData {
```

CSS Klassen werden nicht in Camel Case geschrieben sondern mit Bindestrich getrennt.
```suggestion
.no-data {
```